proto/integration.proto

```diff
@@ -447,4 +447,12 @@ message ForkresolutionConfigRequest{
 
 service ForkresolutionService{
     rpc CompareTwoBranches(TwoBranchesRequest) returns (CompareReply) {}
+}
+
+message EmptyReply{
+}
+
+service PopService{
```

Comment:
What is so special about this service? Why is it separate?

Reply:
we don't need to use most of current api calls. this service is 1x1 api to btc's service.